business/workloads_test.go

```diff
@@ -456,17 +457,99 @@ func TestGetPodLogs(t *testing.T) {
 	assert.Equal("WARN Fake Warning Entry", podLogs.Entries[1].Message)
 	assert.Equal("WARN", podLogs.Entries[1].Severity)
 
-	assert.Equal("2018-01-02T04:34:28+00:00", podLogs.Entries[2].Timestamp)
-	assert.Equal(int64(1514867668), podLogs.Entries[2].TimestampUnix)
+	assert.Equal("2018-01-02T05:34:28+00:00", podLogs.Entries[2].Timestamp)
+	assert.Equal(int64(1514871268), podLogs.Entries[2].TimestampUnix)
 	assert.Equal("Log Entry Without Severity", podLogs.Entries[2].Message)
 	assert.Equal("INFO", podLogs.Entries[2].Severity)
 
-	assert.Equal("2018-01-02T04:34:28+00:00", podLogs.Entries[3].Timestamp)
-	assert.Equal(int64(1514867668), podLogs.Entries[3].TimestampUnix)
+	assert.Equal("2018-01-02T06:34:28+00:00", podLogs.Entries[3].Timestamp)
+	assert.Equal(int64(1514874868), podLogs.Entries[3].TimestampUnix)
 	assert.Equal("error Log Entry With LowerCase Severity", podLogs.Entries[3].Message)
 	assert.Equal("ERROR", podLogs.Entries[3].Severity)
 }
 
+func TestGetPodLogsTailLines(t *testing.T) {
```

Comment:
I didn't look at this test but it should not have been passing given the above impl issue.  perhaps its expectations were wrong.